nifi-nar-bundles/nifi-websocket-bundle/nifi-websocket-services-jetty/src/main/java/org/apache/nifi/websocket/jetty/JettyWebSocketClient.java

```diff
@@ -142,10 +142,10 @@ public void startClient(final ConfigurationContext context) throws Exception{
         client.start();
         activeSessions.clear();
 
-        webSocketUri = new URI(context.getProperty(WS_URI).getValue());
-        connectionTimeoutMillis = context.getProperty(CONNECTION_TIMEOUT).asTimePeriod(TimeUnit.MILLISECONDS);
+        webSocketUri = new URI(context.getProperty(WS_URI).evaluateAttributeExpressions().getValue());
```

Comment:
It looks like you're still using `expressionLanguageSupported(boolean)` in this PR. Please rebase off of the latest master and add change it `expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)`.